index.html

```diff
@@ -50,6 +50,264 @@ <h1>I'm Matunda. Glad to see you!</h1>
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
+
+        </div>
+
+
+      </div>
+    </section>
+    <!--About Me Section-->
+    <section>
+      <div class="about">
+        <div class="top">
+          <h1>About Me</h1>
+          <div class="message">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+          <div class="social-media">
+            <p class="social-media-text">LET'S CONNECT</p>
+            <div class="social-media-icons">
+              <ul>
+              <li><a href="#"><img src="./images/twitter icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Linkedin icon.svg"></a></li>
+              <li><a href="#"><img src="./images/M icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Github icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Chill icon.svg"></a></li>
+            </ul>
+            </div>
+           
+          </div>
+          <div class="resume">
+            <!-- <div class="action"> -->
+              <div class="get-resume-btn">
+                <p>Get my resume</p>
+              </div>
+            
+
+          </div>
+        </div>
+        <ul class="skills-list">
+          <div class="languages">
+            <div class="tab-title">
+              
+              <div class="icon">
+                
+                <img src="./images/Icon - Chevron - Down - Larger.svg" alt="" class="drop-down">
+                
+              </div>
+              <li>Languages</li>
+            </div>
+            <div class="language-list">
+              <div class="js-frame">
+                <p>Javascript</p>
+                <img src="./images/Ellipse 1.svg">
+                
+              </div>
+              <div class="js-frame">
+                <p>HTML</p>
+                <img src="./images/HTML.svg">
+                
+              </div>
+              <div class="js-frame">
+                <p>CSS</p>
+                <img src="./images/CSS.svg">
+                
+              </div>
+            </div>
+          </div>
+          <div class="framework-skill">
+            
+            <div class="arrow-btn">
+              <img src="./images/Icon - Chevron - Right - Larger.svg">
+            </div>
+            <li>Frameworks</li>
+            <img class="separator" src="./images/Separator Bottom.svg">
+          </div>
+          <div class="framework-skill">
+            
+            <div class="arrow-btn">
+              <img src="./images/Icon - Chevron - Right - Larger.svg">
+            </div>
+            <li>Skills</li>
+            <img class="separator" src="./images/Separator Bottom.svg">
+          </div>
```

Comment:
- [ ] As I mentioned in the previous comments, please wrap each `<div class="framework-skill">` in a `li` tag.

Reply:
rearranged the list items as guided